hawc/apps/udf/constants.py

```diff
@@ -0,0 +1,5 @@
+from django.db import models
+
+
+class ContentTypeChoices(models.IntegerChoices):
```

Comment:
I think it may be better to just filter the ContentType queryset in the form; we can add additional models over time.

```python

from django.db.models import Case, F, Q, QuerySet, Value, When
from django.db.models.functions import Concat

ContentType.objects.annotate(
   full_name=Concat(F('app_label'), Value('.'), F('model'))
).filter(
   full_name__in=['animal.endpoint', 'study.study']
)
```